docs/kits.md

```diff
@@ -134,6 +134,10 @@ The following additional options may be specified:
 
 > A JSON object of key-value pairs specifying additional environment variables to be defined when using this kit.
 
+`environmentSetupScript`
+
+> A script that is evaluated to specify additional environment variables to be defined when using this kit. Uses `call` on Windows and `source` in `bash` otherwise.
```

Comment:
@bobbrow do you think we can use this description from the kit schema:
`The absolute path to a script that modifies/adds environment variables for the Kit.`

Reply:
Done.